### DISCUSSION TITLE
Why do I renew emailjs but when I send it, it keeps saying that emailjs has been sent?

### DISCUSSION THREAD

**Discussion by namtran0928** [emailjs-com/help#102] [Q&A]
Why do I renew emailjs but when I send it, it keeps saying that emailjs has been s
![Annotation 2024-04-23 181652](https://github.com/emailjs-com/help/assets/158240595/8a6a5318-7c8c-493b-a0df-a63a9c78fd45)
![Annotation 2024-04-23 181710](https://github.com/emailjs-com/help/assets/158240595/92c514a3-5793-42bf-925d-d0c3c49dc65e)
ent?

**Comment by xr0master** [comment]
Please contact the support via the dashboard, and they will check it.
